tests/features/content_type.feature

```diff
@@ -71,20 +29,3 @@ Feature: Lightning Content Types
     When I visit "/admin/content"
     Then "Action" should not have a "node_publish_action" option
     And "Action" should not have a "node_unpublish_action" option
-
-  @6ef8b654
-  Scenario: Describing a view mode
```

Comment:
Thanks to view_mode_test, I was able to merge this test with another, existing Behat test.